features/tree/handlers.go

```diff
@@ -74,28 +87,36 @@ func (f Feature) GetTree(c *gin.Context) {
 			select descendant as object_id from Paths 
 			where ancestor = ? and path_length != 0 and path_length <= ?
 		);`, rootObjectId, depth)
-
 	if err != nil {
 		c.AbortWithError(http.StatusBadRequest, err)
 		return
 	}
 
+	// ObjectRow represents a row in the object database table
 	type ObjectRow struct {
-		ObjectId       uint64
-		ParentObjectId uint64
-		ObjectKey      string
-		SizeBytes      uint64
-		ObjectType     string
-		TakenAtSec     sql.NullInt64
-		ObjectsInside  sql.NullInt64
-		LockedUntilSec sql.NullInt64
+		ObjectId       uint64        `json:"objectId"                 example:"123"`
+		ParentObjectId uint64        `json:"parentObjectId"           example:"456"`
+		ObjectKey      string        `json:"objectKey"                example:"object_key"`
+		SizeBytes      uint64        `json:"sizeBytes"                example:"102400"`
+		ObjectType     string        `json:"objectType"               example:"image"`
+		TakenAtSec     sql.NullInt64 `json:"takenAtSec,omitempty"     example:"1630342017"`
+		ObjectsInside  sql.NullInt64 `json:"objectsInside,omitempty"  example:"2"`
+		LockedUntilSec sql.NullInt64 `json:"lockedUntilSec,omitempty" example:"1630343017"`
```

Comment:
I think we don't need `omitempty` for these fields. Here is why:
1. What will be the API response if these fields are empty? I think it won't include `takenAtSec`, `objectsInside`, `lockedUntilSec` into JSON. How should a client handle sometimes missing fields? I think it will be tedious. Personally, I prefer keys always present in the API response. This makes parsing easier.
2. Technically, `0` is a correct value for these fields, especially for `objectsInside`. We must not skip `objectsInside` fields if it is `0`, because there are cases when a folder doesn't have any objects inside.

Todo:
- remove `omitempty`